template/scss/themes/litera/_custom.scss

```
@@ -1,6 +1,18 @@
 .badge {
- padding: .4em;
+  padding: .4em;
```

Era `em` mesmo ou `rem`?

```suggestion
  padding: .4rem;
```